### PR TITLE
Add Traditional Chinese localization

### PR DESCRIPTION
**Pull request by ethanhuang13**
## What's Changed

- [x] Added Traditional Chinese Localization File
- [ ] Localize strings (WIP)

## Note

My localization is based on [Apple Localization Terms Glossary](https://applelocalization.com/macos) and myself as a native speaker of Chinese.

### CODE REVIEW

File: Xcodes/Resources/zh-Hant.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+"Menu.CheckForUpdates" = "檢查更新項目⋯";
+"Menu.Acknowledgements" = "Xcodes 版權宣告";
+"Menu.GitHubRepo" = "Xcodes GitHub Repo";
+"Menu.ReportABug" = "回報錯誤";
+"Menu.RequestNewFeature" = "請求新功能";
+
+// Common
+"Install" = "安裝";
+"InstallDescription" = "安裝這個版本";
+"RevealInFinder" = "顯示於Finder";
```
Comment (pofat) [inline Xcodes/Resources/zh-Hant.lproj/Localizable.strings:12]: There should be a space between “顯示於” and “Finder”.

File: Xcodes/Resources/zh-Hant.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+// Updates Preference Pane
+"Updates" = "更新";
+"Versions" = "版本";
+"AutomaticInstallNewVersion" = "Automatically install new versions of Xcode";
+"IncludePreRelease" = "Include prerelease/beta versions";
+"AppUpdates" = "Xcodes.app Updates";
+"CheckForAppUpdates" = "自動檢查更新項目";
+"CheckNow" = "立即檢查";
+"LastChecked" = "上一次檢查：：%@";
```
Comment (pofat) [inline Xcodes/Resources/zh-Hant.lproj/Localizable.strings:71]: Extra colon?

**Comment by MattKiazyk** [comment]
hey @ethanhuang13 how is this PR coming along? Is it ready for review? Looking at merging all the new languages and making a release.

**Comment by ethanhuang13** [comment]
Hi @MattKiazyk 
It's not ready yet. I won't finish it until the following week. Please ignore it if you are going to release a new version.

**Comment by ethanhuang13** [comment]
Personally, I will keep using the English version of Xcodes. Therefore I've decided not to localize it.

Someone could take over. Announced to the Traditional Chinese iOS community on [Twitter](https://twitter.com/ethanhuang13/status/1532986286306639873?s=21&t=dbqG83ehsdidJdnttA8tQw).

**Pull request closed without merging**